tests/testHtmIndex.py

```diff
@@ -30,21 +32,20 @@
 
 import numpy as np
 
-import lsst.utils
 import lsst.afw.table as afwTable
 import lsst.afw.geom as afwGeom
 import lsst.afw.coord as afwCoord
 import lsst.daf.persistence as dafPersist
 from lsst.meas.algorithms import IngestIndexedReferenceTask, LoadIndexedReferenceObjectsTask, \
     LoadIndexedReferenceObjectsConfig, getRefFluxField
```

Comment:
Use braces for line wrapping?
https://developer.lsst.io/coding/python_style_guide.html?highlight=print%20statements#python-s-implied-continuation-inside-parens-brackets-and-braces-should-be-used-for-wrapped-lines